vllm/executor/ray_gpu_executor.py

```diff
@@ -382,8 +436,17 @@ async def _run_task_with_lock(task, lock, *args, **kwargs):
         return results[-1]
 
     async def _start_worker_execution_loop(self):
+        assert not USE_SPMD_WORKER, (
+            "worker loop is disabled for VLLM_USE_SPMD_WORKER=1")
```

Comment:
you don't need to put any `assert not USE_SPMD_WORKER`, but just surround the part of new code under `if USE_SPMD_WORKER`.

Reply:
Yeah, the `if` is done at the caller.
The `assert` is a sanity check to catch any invalid uses.